examples/deployer/src/createNewWallet.ts

```diff
@@ -1,36 +1,21 @@
-import {
-  TurnkeyClient,
-  createActivityPoller,
-  TurnkeyActivityError,
-} from "@turnkey/http";
-import { ApiKeyStamper } from "@turnkey/api-key-stamper";
+import { Turnkey as TurnkeySDKServer } from "@turnkey/sdk-server";
```

Comment:
Much cleaner 😍 - I think we should continue leaving the lower level http functions for our authentication examples (email auth, otp auth, oauth) for now though (which is what was done here 🤝)

Reply:
gut feeling is we should start to migrate everything to the new libraries, but maybe it'd be useful to leave one or two using the "legacy" suite of libraries